ComputeCutVarPromptFrac.py

```diff
@@ -181,6 +186,14 @@
     gFDFracNbVsCut, cFrac = [], [], [], [], [], [], []
 hCorrMatrixCutSets, cCorrMatrix = [], []
 
+if cutSetCfg['linearplot']['enable']:
```

Comment:
Since you added an argument in the yaml config, could you please update these example config files [config_Dplus_PromptFrac_pp5TeV.yml](https://github.com/DmesonAnalysers/DmesonAnalysis/blob/master/configfiles/datadrivenfprompt/config_Dplus_PromptFrac_pp5TeV.yml), [config_Ds_PromptFrac_pp5TeV.yml](https://github.com/DmesonAnalysers/DmesonAnalysis/blob/master/configfiles/datadrivenfprompt/config_Ds_PromptFrac_pp5TeV.yml), and [config_Ds_FDFrac_MC_ITS2.yml](https://github.com/DmesonAnalysers/DmesonAnalysis/blob/master/configfiles/datadrivenfprompt/config_Ds_FDFrac_MC_ITS2.yml) adding the additional argument? (set to False or True as you prefer)

Reply:
Hi @fgrosa! I reply here once for all: I'll surely upload also a config-file!